src/TrajectoryPlanner/src/TrajectoryGenerator.cpp

```diff
@@ -508,6 +521,31 @@ bool TrajectoryGenerator::getDCMPositionTrajectory(std::vector<iDynTree::Vector2
     return true;
 }
 
+bool TrajectoryGenerator::getDCMPositionTrajectoryAdjusted(std::vector<iDynTree::Vector2>& DCMPositionTrajectory)
+{
+    DCMPositionTrajectory = m_dcmGenerator->getDCMPosition();
+    return true;
+}
+
+bool TrajectoryGenerator::getDCMVelocityTrajectoryAdjusted(std::vector<iDynTree::Vector2>& DCMVelocityTrajectory)
+{
+    DCMVelocityTrajectory = m_dcmGenerator->getDCMVelocity();
```

Comment:
You should add a check on whether trajectories have been computed using ``isTrajectoryComputed()``.

Reply:
Done!